### ISSUE TITLE
version 1.4 ignore header

### ISSUE THREAD

**Issue by Veiintc** [tophcodes/guacamole-common-js#16]
![image](https://user-images.githubusercontent.com/32948063/157005271-c8ddb2a2-90a9-4e82-9ba3-d8f2b2fa9eaa.png)
this is guacamole-common-js@1.4.0
but I see github, it should three args
and I install guacamole-common-js@1.3.1,it had three args

**Comment by padarom** [comment]
Mh I'm unsure how this happened. The distributable files were regenerated after I updated from upstream, so not sure how it ended up with just one argument. I just regenerated them again and now everything seems in order:

```
Guacamole.HTTPTunnel = function(tunnelURL, crossDomain, extraTunnelHeaders) {
```

I just published these as a patch `1.4.0-a`. Thanks for pointing this out and apologies for the inconvenience!

**Comment by Veiintc** [comment]
Thank you for your quick reply

**Comment by TaeHyunChi** [comment]
The extraTunnelHeaders is not sent while making a request in 1.4.x 
so I did fix it by restoring version 1.3.1
